playbooks/setup_k8s_cluster.yml

```diff
@@ -59,47 +41,6 @@
         src: /vagrant/playbooks/k3s_token
       register: k3s_token
 
-    - name: Ensure .kube directory exists
-      file:
-        path: /home/vagrant/.kube
-        state: directory
-        mode: '0755'
-        owner: vagrant
-        group: vagrant
-
-    - name: copy ./k3s.yaml from local machine to worker nodes
-      copy:
-        content: "{{ lookup('file', './k3s.yaml') }}"
-        dest: /home/vagrant/.kube/config
-
-    - name: Change ownership of kubeconfig
-      file:
-        path: /home/vagrant/.kube/config
-        owner: vagrant
-        group: vagrant
-        mode: '0644'
-
-    - name: set correct api server
-      replace:
-        path: /home/vagrant/.kube/config
-        regexp: 'server: https://127.0.0.1:6443'
-        replace: 'server: https://192.168.56.10:6443'
-
     - name: Join worker nodes to the cluster
       shell: |
-        curl -sfL https://get.k3s.io | K3S_URL=https://{{ hostvars['controller'].ansible_host }}:6443 K3S_TOKEN={{ k3s_token['content'] | b64decode | regex_replace('\n', '')}} sh -s - agent --node-ip={{ ansible_host }}
-
-- name: Setup cluster
-  hosts: controller
-  become: yes
-  tasks:
-    - name: Export config
-      shell: |
-        export KUBECONFIG=~/.kube/config
-
-    - name: Apply Kubernetes yml files
-      shell: |
-        kubectl apply -f ./../vagrant/kubernetes/environment.yml
-        kubectl apply -f ./../vagrant/kubernetes/app.yml
-        kubectl apply -f ./../vagrant/kubernetes/model-service.yml
-        kubectl apply -f ../../vagrant/kubernetes/ingress.yml
```

Comment:
Do we really want to exclude this from the playbooks?

Reply:
Yeah I guess, but then we need to update the readme what exactly needs to be applied and installed

Reply:
yeah but also just for us

Reply:
ill put it back then

Reply:
too late ;p, ill still add the above to the README for now, but in the future a user should not have to do that